### DISCUSSION TITLE
Why "GitHub Actions is currently disabled for this repository." occurs? & How can it be reenabled?

### DISCUSSION THREAD

**Discussion by zhuyuqing** [community/community#46975] [Actions]
### Select Topic Area

Product Feedback

### Body

Today, our repository is pop up with "GitHub Actions is currently disabled for this repository." 

Why would this happy? How can it be reenabled? 

Thank you!

**Comment by chris-c-thomas** [comment]
Were you using Actions before?

Click on the Actions tab from within the repository and you can turn back on specific workflows if one got disabled.
